assets/src/edit-story/components/form/media.js

```diff
@@ -181,7 +181,7 @@ MediaInput.propTypes = {
   disabled: PropTypes.bool,
   size: PropTypes.number,
   circle: PropTypes.bool,
-  ariaLabel: PropTypes.string.isRequired,
+  ariaLabel: PropTypes.string,
```

Comment:
Why remove the required though?

Reply:
Probably because there are lots of prop types (missing prop) warnings when running tests.

The correct fix would of course be to pass the required prop (as you've commented below). I addressed this in #2254.

Reply:
My latest change aligns ariaLabel with other props `title, buttonInsertText` (not required - have defaults).

Reply:
Alright, there was actually a PR at some point reworking most of the labels to do the opposite: made the media label required and also removed a bunch of default values from most of the inputs -- to enforce having proper aria labels and not generic ones which aren't really informative quite often.

Reply:
(The goal being actually providing informative labels as opposed to just avoiding console errors)